packages/plexus/src/zoom/condenseIcon.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as React from 'react';
+
+// SVG is from Material Design Icons:
+// https://github.com/google/material-design-icons/blob/224895a86501195e7a7ff3dde18e39f00b8e3d5a/av/svg/production/ic_shuffle_48px.svg
+// LICENSE: Apache-2.0
```

Comment:
@yurishkuro I followed the example in `src/zoom/resetIcon.tsx`, is this the correct way to note the LICENSE followed?

Reply:
it's good